### PR TITLE
add Test DTO ProxyConfigHolder

### PR DESCRIPTION
**Pull request by iatsenkoo**

### CODE REVIEW

**Review by artemkedyk — DISMISSED** [review]

File: src/test/java/executor/service/model/ProxyConfigHolderTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package executor.service.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+public class ProxyConfigHolderTest {
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:10]: First of all, let's make all the fields of ProxyConfigHolder non package-private to avoid their non-controlled changing

Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:10]: Let's add default constructor cause it is impossible to create an empty object, for example for testing goals

File: src/test/java/executor/service/model/ProxyConfigHolderTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    private static ProxyConfigHolder proxyConfigHolder2;
+
+    @Before
+    public void setUp() {
+        proxyConfigHolder = new ProxyConfigHolder(proxyNetworkConfig,proxyCredentials);
+        proxyConfigHolder2 = new ProxyConfigHolder(proxyNetworkConfig,proxyCredentials);
+    }
+
+    @Test
+    public void equalsTest(){
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:23]: style: add a space before the opening curly brace (for all tests)

File: src/test/java/executor/service/model/ProxyConfigHolderTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        assertEquals(proxyConfigHolder,proxyConfigHolder2);
+    }
+
+    @Test
+    public void hashCodeTest(){
+        assertEquals(proxyConfigHolder.hashCode(),proxyConfigHolder2.hashCode());
+    }
+
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:31]: Let's add tests for empty / different objects to check 'equals' and 'hashCode' methods

File: src/test/java/executor/service/model/ProxyConfigHolderTest.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    @Test
+    public void setGetProxyNetworkConfigTest(){
+        proxyNetworkConfig.setHostname("localHost2");
+        proxyNetworkConfig.setPort(441);
+        proxyConfigHolder.setProxyNetworkConfig(proxyNetworkConfig);
+
+        assertEquals(proxyConfigHolder.getProxyNetworkConfig().getHostname(),"localHost2");
+        assertEquals(proxyConfigHolder.getProxyNetworkConfig().getPort().intValue(),441);
+    }
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:40]: I'd recommend to use a variables to set hostName and port.
If you need to make some changes in the test, now you will update the test in two lines for every value
(for example, line 35 and 39).
And you will make the only change when using variable on the level of current class

File: src/test/java/executor/service/model/ProxyConfigHolderTest.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    @Test
+    public void setGetProxyCredentialsTest(){
+        proxyCredentials.setUsername("login");
+        proxyCredentials.setPassword("new pass");
+        proxyConfigHolder.setProxyCredentials(proxyCredentials);
+
+        assertEquals(proxyConfigHolder2.getProxyCredentials().getUsername(),"login");
+        assertEquals(proxyConfigHolder2.getProxyCredentials().getPassword(),"new pass");
+
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyConfigHolderTest.java:50]: The same recommendation:
Let's use a variables to set username and password.

**Review by maksymivanov2510 — APPROVED** [review]
Looks good! Let's merge the branch
